ISLE/dummy_controller_map.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
Let's please use consistent include guards (the one original isle uses)

Reply:
the ifndef and define method? sure